<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customindoor</class>
 <widget class="QDialog" name="customindoor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#customindoor {
   border-image:url(:/images/images/background.PNG);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#002244;</string>
       </property>
       <property name="text">
        <string>Hunza Cultural Complex: Immerse in Hunza's heritage with exhibits and workshops   ---PKR 500</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#002244;</string>
       </property>
       <property name="text">
        <string>Altit Fort Museum: Step back in time and learn about Hunza royalty and Silk Road trade  --- PKR 750</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#002244;</string>
       </property>
       <property name="text">
        <string>Local Handicraft Center: Discover and support local artisans  ---PKR 1200</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_4">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#002244;</string>
       </property>
       <property name="text">
        <string>Traditional Music &amp; Dance Show: Immerse in local culture with music and dance   ---PKR 600</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_5">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#002244;</string>
       </property>
       <property name="text">
        <string>Gilgit Serena Hotel Afternoon Tea: Experience elegance and chai at Serena Hotel --PKR 2000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_6">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#002244;</string>
       </property>
       <property name="text">
        <string>Hunza Food Cooking Class: Master Hunza cuisine and enjoy your creations  --PKR 2000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_7">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#002244;</string>
       </property>
       <property name="text">
        <string>Shigar Polo Grounds: Witness the unique Shigar polo game played on horseback    ---PKR 100</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextbtn">
       <property name="styleSheet">
        <string notr="true">padding: 10px;
border-radius:10px;
background-color: rgb(89, 170, 163);
color:rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>22</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#002244</string>
     </property>
     <property name="text">
      <string>Select Indoor Activitites (Maximum Five)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
